### PR TITLE
Home page and search bar skeleton - Riaz

### PR DESCRIPTION
**Pull request by riazahmed01**
Changes:

- Removed next.js template from home page
- Added SearchBar.js component
- Home page now renders search bar and recipe view components

### CODE REVIEW

File: src/app/page.js [hunk 1]
```diff
@@ -1,113 +1,104 @@
...
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Learn about Next.js in an interactive course with&nbsp;quizzes!
-          </p>
-        </a>
+  return (
+      <div className="">
+        <div className="homeCover text-center height: md:h-[400px] h-auto">
```
Comment (NikhilPrajith) [inline src/app/page.js:76]: nit: Can we make the min-height maybe 550px or 500px, it might look better on smaller screens

File: src/app/page.js [hunk 2]
```diff
@@ -1,113 +1,104 @@
...
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
+        <div className='lg:m-28 md:m-18 mt-8'>
+          {/* Load some recommendations for home page*/}
+          <div>
+            <h1 className='px-6 mb-10 text-center lg:text-left text-2xl lg:text-4xl md:text-3xl font-semibold text-black'>
```
Comment (NikhilPrajith) [inline src/app/page.js:94]: nit: So we want it to be consistent, so I used 30px for the title on the explore page. Either we can use that here as well or something else that you can mention in the notes file and I will fix the title on the explore page.
You can use style={{fontSize:'30px'}}

**Review by NikhilPrajith — APPROVED** [review]
LGTM

**Comment by azuhayer** [comment]
LGTM

**Review by azuhayer — APPROVED** [review]

**Review by jchen22002 — APPROVED** [review]
PR Approved! Looks good, go ahead and merge when ready.

**Pull request merged**